server/plugin.go

```diff
@@ -49,21 +49,26 @@ func (p *Plugin) OnActivate() error {
 	teamSplit := split[0]
 	channelSplit := split[1]
 
-	team, err := p.API.GetTeamByName(teamSplit)
-	if err != nil {
-		return err
+	team, appErr := p.API.GetTeamByName(teamSplit)
+	if appErr != nil {
+		return appErr
 	}
 	p.TeamID = team.Id
 
-	user, err := p.API.GetUserByUsername(p.configuration.Username)
+	botID, err := p.Helpers.EnsureBot(&model.Bot{
+		Username:    "aws-sns",
+		DisplayName: "AWS SNS Plugin",
+		Description: "A bot account created by the plugin AWS SNS",
+	},
+		plugin.ProfileImagePath("assets/profile.png"),
```

Comment:
The call to `EnsureBot` here expects there to be a file `assets/profile.png` in this repo. In this PR, I think we should include a `profile.png` picture like the one [here](https://cdn2.iconfinder.com/data/icons/amazon-aws-stencils/100/App_Services_copy_Amazon_SNS-512.png).

Is this working locally for you @vesari?

Reply:
Or use the svg icon that is already there
```suggestion
		plugin.ProfileImagePath("assets/icon.svg"),
```

Reply:
I saw the icon file and thought it was a generic placeholder since it doesn't look directly affiliated with SNS, but it looks like it fits the theme of SNS. It could even be an official image I don't recognize :sweat_smile: 

![image](https://user-images.githubusercontent.com/6913320/76867664-37b29b80-683c-11ea-8f16-67bbaa5f6d56.png)